cs/src/Contracts/TunnelConstraints.cs

```diff
@@ -306,6 +306,10 @@ public static bool IsValidTunnelName(string tunnelName)
         {
             return false;
         }
+        if (tunnelName.EndsWith("-inspect"))
```

Comment:
Use overload with string comparison `OrdinalIgnoreCase`. The tunnel names are use in host name, and that is case-insensitive.

Reply:
added